ckanext/data_qld/templates/reporting/snippets/admin_report.html

```diff
@@ -38,29 +38,30 @@ <h2>Organisation: {{ org_title }}</h2>
             <th width="25%">Figure</th>
         </thead>
         <tbody>
+            {% set metric_link = '/dashboard/reporting/datasets/{org_id}/de-identified-datasets?report_type={report_type}'.format(org_id=org_id, report_type=report_type) %}
```

Comment:
This is incorrect, they aren't all the same link. One is de-identified datasets, the next is overdue datasets, etc.

Reply:
Oops, fixed the link.